web_monitoring/tests/test_diffing_server_exc_handling.py

```diff
@@ -166,6 +166,26 @@ def test_check_cors_headers(self):
         assert response.headers.get('Access-Control-Allow-Headers') == 'x-requested-with'
         assert response.headers.get('Access-Control-Allow-Methods') == 'GET, OPTIONS'
 
+    # The processing of the access_control_allow_origin_header to produce
+    # ALLOWED_ORIGINS happens inside the diffing_server module and not on web
+    # app init. As a result, we need to mock patch both of them here to make
+    # the unit test work.
```

Comment:
See my notes above about supporting `*` — I think it might make sense to not have the handler code ever look at `access_control_allow_origin_header` and instead only examine the parsed `ALLOWED_ORIGINS` set. If that’s the case, you’d only need to patch one thing here.

Reply:
Updated unit test.